grails-app/controllers/au/org/ala/profile/api/ApiController.groovy

```diff
@@ -106,6 +107,41 @@ class ApiController extends BaseController {
         }
     }
 
+    @Path("/api/opus/list")
+    @Operation(
+            summary = "Get all collection (opus) details",
+            operationId = "/api/opus/list",
+            method = "GET",
+            responses = [
+                    @ApiResponse(
+                            responseCode = "200",
+                            content = @Content(
+                                    mediaType = "application/json",
+                                    array = @ArraySchema(
+                                            schema = @Schema(
+                                                    implementation = OpusResponse.class
+                                            )
+                                    )
+                            )
+                    ),
+                    @ApiResponse(responseCode = "400",
+                            description = "opusId is a required parameter"),
+                    @ApiResponse(responseCode = "403",
+                            description = "You do not have the necessary permissions to perform this action."),
+                    @ApiResponse(responseCode = "405",
+                            description = "An unexpected error has occurred while processing your request."),
+                    @ApiResponse(responseCode = "404",
+                            description = "Collection not found"),
+                    @ApiResponse(responseCode = "500",
+                            description = "An unexpected error has occurred while processing your request.")
+            ]
+    )
+    def getListCollections () {
+       List opus = profileService.getOpusList() as List
```

Comment:
Reuse `profileService.getOpus()`. No need to implement a custom code in profile-hub or profile-service.